omnigibson/robots/a1.py

```diff
@@ -0,0 +1,221 @@
+                the object state instance constructor.
+            control_freq (float): control frequency (in Hz) at which to control the object. If set to be None,
+                we will automatically set the control frequency to be at the render frequency by default.
+            controller_config (None or dict): nested dictionary mapping controller name(s) to specific controller
+                configurations for this object. This will override any default values specified by this class.
+            action_type (str): one of {discrete, continuous} - what type of action space to use
+            action_normalize (bool): whether to normalize inputted actions. This will override any default values
+                specified by this class.
+            reset_joint_pos (None or n-array): if specified, should be the joint positions that the object should
+                be set to during a reset. If None (default), self._default_joint_pos will be used instead.
+                Note that _default_joint_pos are hardcoded & precomputed, and thus should not be modified by the user.
+                Set this value instead if you want to initialize the robot with a different rese joint position.
+            obs_modalities (str or list of str): Observation modalities to use for this robot. Default is ["rgb", "proprio"].
+                Valid options are "all", or a list containing any subset of omnigibson.sensors.ALL_SENSOR_MODALITIES.
+                Note: If @sensor_config explicitly specifies `modalities` for a given sensor class, it will
+                    override any values specified from @obs_modalities!
+            proprio_obs (str or list of str): proprioception observation key(s) to use for generating proprioceptive
+                observations. If str, should be exactly "default" -- this results in the default proprioception
+                observations being used, as defined by self.default_proprio_obs. See self._get_proprioception_dict
+                for valid key choices
+            sensor_config (None or dict): nested dictionary mapping sensor class name(s) to specific sensor
+                configurations for this object. This will override any default values specified by this class.
+            grasping_mode (str): One of {"physical", "assisted", "sticky"}.
+                If "physical", no assistive grasping will be applied (relies on contact friction + finger force).
+                If "assisted", will magnetize any object touching and within the gripper's fingers.
+                If "sticky", will magnetize any object touching the gripper's fingers.
+            kwargs (dict): Additional keyword arguments that are used for other super() calls from subclasses, allowing
+                for flexible compositions of various object subclasses (e.g.: Robot is USDObject + ControllableObject).
+        """
+        self._hand_part_names = [11, 12, 13, 14, 21, 22, 31, 32, 41, 42, 51, 52]
```

Comment:
are these names or idxs? Should we just explicitly have all the names? (or programmatically generate the list, i.e.: `[f"link_{idx} for idx in [11, 12, ...]]`?